### ISSUE TITLE
Confirming output of bed_autoSVD

### ISSUE THREAD

**Issue by kwerwath** [privefl/paper4-bedpca#2]
Hi!

I calculated PCs for a dataset using bed_autoSVD and then wrote the PC values to a table using this code: 
```
obj.svd2 <- bed_autoSVD(...)
PC <- predict(obj.svd2)
write.table(PC)
```
(Omitting arguments to bed_autoSVD for brevity)

The output file has PC values, but no subject IDs. Is it fair to assume that the PCs are in the same order as the subject IDs in the .fam file inputted to bed_autoSVD?

Thank you so much!

Katy Werwath

**Comment by privefl** [comment]
Yes, this should be in the same order. Or in the order of `ind.row` if you specify this parameter to `bed_autoSVD()`.
